test/images/sun.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generated by IcoMoon.io -->
```

Comment:
Is this SVG free? Can you generate your own svg file?

Reply:
Yes, it is free, but I will generate my own svg anyway